all-actors/src/main/java/org/sunbird/Application.java

```diff
@@ -0,0 +1,43 @@
+package org.sunbird;
+
+import akka.actor.ActorRef;
+import org.sunbird.actor.core.ActorCache;
+import org.sunbird.actor.core.ActorService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * this class is used to instantiate the actor system and open saber.
+ * @author Amit Kumar
+ */
+public class Application {
+    private final static String actorSystemName = "thisActorSystem";
```

Comment:
chnage actor name here. it should be relevent to code base and same chnages will be applicable under application.conf

Reply:
done